units/reauthorize_user.go

```diff
@@ -127,31 +127,27 @@ func (j *reauthorizationJob) Run(ctx context.Context) {
 	// Okta has expired, in which case they should be logged out, so that they
 	// are forced to log in to get a new refresh token.
 	if err != nil && strings.Contains(err.Error(), "invalid_grant") && strings.Contains(err.Error(), "The refresh token is invalid or expired.") {
-		grip.Debug(message.WrapError(err, message.Fields{
+		grip.Info(message.WrapError(err, message.Fields{
```

Comment:
Is this worth making `info`?

Reply:
It doesn't log that often and it's useful to know when someone is logged out by a background job. In case of actual error diagnosing, I would prefer to see this in case we suppress debug logs in production.